backends-common/rabbitmq/src/main/java/org/apache/james/backends/rabbitmq/QueueArguments.java

```diff
@@ -70,6 +70,11 @@ public Builder deliveryLimit(long deliveryLimit) {
             return this;
         }
 
+        public Builder classicQueueVersion(int version) {
+            arguments.put("x-queue-version", version);
```

Comment:
Its inter-compatibility with the older James versions is good.
This works (be skipped) with old RabbitMQ version 3.8.x and 3.9.x for example.